noxfile.py

```diff
@@ -28,6 +28,8 @@
 if shutil.which("conda"):
     nox.options.sessions.append("conda_tests")
 
+nox.options.sessions = []
```

Comment:
Somehow this debugging statement is now back?

```suggestion
```

I assume this wasn't your intention as `nox.options.sessions` is assigned prior to this line.

Reply:
Oops, thanks!